db/scripts/01_cassandra_queue.cql

```diff
@@ -13,7 +14,7 @@ CREATE TABLE queue (
 );
 
 CREATE TABLE queue_size (
```

Comment:
hmm, looksl ike youre' right. though I'm not clear if you can have multiple counters in a single table